spec/system/candidate_signs_up_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+describe 'A candidate signs up to Apply service' do
+  context 'when a new candidate tries to sign in' do
+    before do
+      visit '/'
+      click_on t('application_form.begin_button')
+      fill_in t('authentication.sign_up.email_address.label'), with: 'april@pawnee.com'
+      click_on t('authentication.sign_up.button')
+    end
+
+    it 'can allow a candidate to submit their email' do
+      expect(page).to have_content t('authentication.check_your_email')
+    end
+
+    it 'can sign in a candidate using the magic link' do
+      open_email('april@pawnee.com')
+      current_email.find_css('a').first.click
+
+      expect(page).to have_content 'april@pawnee.com'
+    end
+  end
+
+  context 'when a candidate is not signed in' do
```

Comment:
Not clear what the difference between this context and the above context is

Reply:
The context above has been replaced with some controller tests.